README.md

```diff
@@ -136,6 +136,7 @@ Name                   | Environment Variable Name            | Description
 redis.addr             | REDIS_ADDR                           | Address of the Redis instance, defaults to `redis://localhost:6379`.
 redis.user             | REDIS_USER                           | User name to use for authentication (Redis ACL for Redis 6.0 and newer).
 redis.password         | REDIS_PASSWORD                       | Password of the Redis instance, defaults to `""` (no password).
+redis.password-file         | REDIS_PASSWORD_FILE                       | Password file of the Redis instance to scrape, defaults to `""` (no password file)，only takes effect when `redis.password == ""` . 
```

Comment:
Can you fix the formatting here please?

Reply:
The README could also use a short description on how to use the file, maybe further down in the "Authenticating..." section.